todo_app/api/todoItem.py

```diff
@@ -0,0 +1,57 @@
+from enum import IntEnum
+
+class TodoItemStatus(IntEnum):
+    NOT_STARTED = 1
+    IN_PROGRESS = 2
+    COMPLETE = 3
+
+not_started_text = "Not Started"
+in_progress_text = "In Progress"
+complete_text = "Complete"
+
+def get_string_as_todo_item_status(status_string: str):
+    if status_string == not_started_text:
+        return TodoItemStatus.NOT_STARTED
+    if status_string == in_progress_text:
+        return TodoItemStatus.IN_PROGRESS
+    if status_string == complete_text:
+        return TodoItemStatus.COMPLETE
+    return None
```

Comment:
I prefer your solution for the clear readability, but in case of interest, worth seeing how you could equally do this with a dictionary:
```python
    string_to_status_mapping = {
        not_started_text: TodoItemStatus.NOT_STARTED,
        in_progress_text: TodoItemStatus.IN_PROGRESS,
        complete_text: TodoItemStatus.COMPLETE
    }
    return string_to_status_mapping.get(status_string, None)
```